hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -343,15 +343,15 @@ boolean needsBalance(TableName tableName, BalancerClusterState cluster) {
       sendRejectionReasonToRingBuffer(() -> getBalanceReason(calculatedTotal, calculatedMultiplier),
         costFunctions);
     }
-    if (LOG.isDebugEnabled()) {
-      LOG.debug("{} {}; total cost={}, sum multiplier={}; cost/multiplier to need a balance is {}",
-          balanced ? "Skipping load balancing because balanced" : "We need to load balance",
-          isByTable ? String.format("table (%s)", tableName) : "cluster",
-          total, sumMultiplier, minCostNeedBalance);
-      if (LOG.isTraceEnabled()) {
-        LOG.trace("Balance decision detailed function costs={}", functionCost());
-      }
+    LOG.info("{} {}; total cost={}, sum multiplier={}; cost/multiplier to need a balance is {}",
```

Comment:
What is cost? And multiplier? And how do the relate to whether a cluster is balanced or not?

Reply:
Hmm, right. Need to explain cost /multiplier is the measurement how balanced the cluster for balancer at a scale on [0,1] 0 being the perfectly balanced and 1 is the worst it could be. But the actual value still doesn't seem very intuitive to operator. We can explain if we demand the max of absolute number of unbalanced nodes to be constant, the bigger the cluster, the smaller of this value would be.